### PR TITLE
Fourmolu 13 fixafix

### PR DESCRIPTION
**Pull request by ffaf1**
---
Please include the following checklist in your PR:

* [ ] Patches conform to the [coding conventions](https://github.com/haskell/cabal/blob/master/CONTRIBUTING.md#other-conventions).
* [ ] Any changes that could be relevant to users [have been recorded in the changelog](https://github.com/haskell/cabal/blob/master/CONTRIBUTING.md#changelog).
* [ ] The documentation has been updated, if necessary.
* [ ] Include [manual QA notes](https://github.com/haskell/cabal/blob/master/CONTRIBUTING.md#qa-notes) if your PR relates to cabal-install.

Bonus points for added automated tests!

### CODE REVIEW

**Review by geekosaur — APPROVED** [review]

**Comment by ffaf1** [comment]
Actually CI failing is not that bad, it means fourmolu 0.12 is *not* confused by new lines in the `yaml`. There are still things to iron out (check commit descs), but this should be painless.

**Comment by ulysses4ever** [comment]
Interesting. Did you see https://github.com/haskell/cabal/pull/9095?

**Comment by ulysses4ever** [comment]
No worries. Yours definitely adds good stuff. Are we switching to 0.13? Where is the ultimate source of truth as to the version?

File: fourmolu.yaml [hunk 1]
```diff
@@ -7,8 +7,18 @@ respectful: true # don't be too opinionated about newlines etc.
 haddock-style: single-line # '--' vs. '{-'
 haddock-style-module: single-line
 newlines-between-decls: 1 # number of newlines between top-level declarations
-fixities: []
+fixities:
+  - infixr 0  <?>
+  - infixl 4  <@>
+  - infixl 5  ^^^
+  - infixr 5  <+>
```
Comment (brandonchinn178) [inline fourmolu.yaml:14]: `<+>`, `<@>`, and `<.>` are all from upstream packages, right? You shouldnt need to add their fixities here

**Review by brandonchinn178 — COMMENTED** [review]
My PR updates to fourmolu 0.12; this PR should probably block/rebase on my PR

File: fourmolu.yaml (hunk 1 above)
Comment (ffaf1) [reply fourmolu.yaml:14]: If I eliminate `<@>`, the diff gets longer. Is this expected?

**Comment by ffaf1** [comment]
> Are we switching to 0.13?

I don't think so, I think we will stay with 0.12 for a long long time.

I believe there is value in keeping `fourmolu.yaml` updated so we can check how we are diverging and picking the most convenient time to perform a quick, clean, painless update.

File: fourmolu.yaml (hunk 1 above)
Comment (brandonchinn178) [reply fourmolu.yaml:14]: Oh sorry I meant `<?>`

Comment (ffaf1) [reply fourmolu.yaml:14]: https://github.com/haskell/cabal/blob/31ae312de34f848d7c653cbcef75e35d7fbab166/Cabal-syntax/src/Distribution/Compat/Parsing.hs#L212

**Comment by brandonchinn178** [comment]
@ffaf1 This PR should also update `CONTRIBUTING.md` to mention fourmolu 0.13. Can you also update `format.yml` to use `run-fourmolu@v9` and specify the version?
https://github.com/haskell-actions/run-fourmolu/releases/tag/v9

**Comment by ffaf1** [comment]
@brandonchinn178 your pr is the proper one, I believe we are staying on 0.12 for the foreseeable future.

If there is no objection, I will close this one for now!

**Pull request closed without merging**